### DISCUSSION TITLE
Rounding issue with 'cvc5.pythonic. fpToSBV ( rm , x , s , ctx = None )'

### DISCUSSION THREAD

**Discussion by Btraube** [cvc5/cvc5#10962] [Q&A]
Hello! 

I am having an issue with cvc5's fpToSBV function. I am trying to convert the output from the fpSqrt instruction by using the fpToSBV function. Below is some inputs and outputs from an ipython terminal demonstrating the issue.

```
fp1 = FPVal(5.5555)

simplify(fp1)
1.388875*(2**2)    # 5.5555

simplify(fpSqrt(RNE(), fp1))
1.1785054094063379*(2**1)   # 2.35701081881

simplify(fpToSBV(RNE(), fpSqrt(RNE(), fp1), BitVecSort(64)))
2

simplify(fpToSBV(RNA(), fpSqrt(RNE(), fp1), BitVecSort(64)))
2

simplify(fpToSBV(RTP(), fpSqrt(RNE(), fp1), BitVecSort(64)))
3

simplify(fpToSBV(RTN(), fpSqrt(RNE(), fp1), BitVecSort(64)))
2

simplify(fpToSBV(RTZ(), fpSqrt(RNE(), fp1), BitVecSort(64)))
2
```

I am getting the correct output from declaring fp1 and from performing a Sqrt on fp1, the issue appears to occur during the fpToSBV conversion. I've guessed that is is related to a rounding issue; however, I am not entirely sure regarding that. I have also tried some of the other rounding modes, but those have all yielded the same result. 

Just to make sure it isnt a size related issue, i also tried the following:

```
fp1 = FPVal(5.5555, FPSort(11, 53))

simplify(fpToSBV(RNE(), fpSqrt(RNE(), fp1), BitVecSort(64)))
2

fp1 = FPVal(5.5555, FPSort(8, 24))

simplify(fpToSBV(RNE(), fpSqrt(RNE(), fp1), BitVecSort(64)))
2
```

Any help with this would be greatly appreciated!  

EDIT: Not sure if this should be in the "Issue" section rather than discussion. Let me know if I should add it over there!

**Comment by aniemetz** [comment]
Duplicate of #10997.
